test/e2e/_roles.js

```diff
@@ -41,6 +41,11 @@ export const pmuUser = Role(home, async t => {
   await page.chooseLocation()
 })
 
+export const pmuUserWithRegions = Role(home, async t => {
```

Comment:
@teneightfive So it turns out that many of our tests use the PMU user, but some need a location to be chosen, and some will need a region to be chosen.

The only place we select a location is within these `_roles` methods.

An alternative to the approach here would be to pass in a "choose location / choose region" variable, which also feels a bit hacky.